_drafts/rails-migrations.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: Rails Migrations
+---
+
+We had a lively conversation today at [Big Nerd Ranch][bnr] about Rails migrations.
+What are they really for?
+How long should you keep them around?
+What about this crazy schema.rb file?!
+TIL WE HAZ OPINIONS!
+
+⚠️ The views of this post are my own.
+This is in no away a representation of our team's view of the subject.
+Quite the contrary, actually.
```

Comment:
I was thinking that, but forgot to say something about it. Yeah, I don't think there was any disagreement about the implications of the two different views. You can definitely say which side you lean more towards personally (unless you're now more confused than when we started!)